x/ccv/provider/keeper/partial_set_security.go

```diff
@@ -91,6 +91,9 @@ func (k Keeper) HandleOptOut(ctx sdk.Context, chainID string, providerAddr types
 // OptInTopNValidators opts in to `chainID` all the `bondedValidators` that have at least `minPowerToOptIn` power
 func (k Keeper) OptInTopNValidators(ctx sdk.Context, chainID string, bondedValidators []stakingtypes.Validator, minPowerToOptIn int64) {
 	for _, val := range bondedValidators {
+		// log the validator
+		k.Logger(ctx).Info("Checking whether to opt in validator because of top N", "validator", val.GetOperator())
```

Comment:
Either change to debug or remove. This might lead to a lot of logging messages. One for every TopN validator.

Reply:
Changing to debug!